util/util.go

```diff
@@ -264,3 +264,12 @@ func MakeRequest(url, method string, reqBody []byte) ([]byte, *http.Response, er
 	}
 	return body, response, nil
 }
+
+func Find(slice []string, val string) (int, bool) {
```

Comment:
@ShahAnuj2610 util for this already exists
https://github.com/appbaseio/arc/blob/dev/util/util.go#L162